https://github.com/linux-nvme/nvme-cli/commit/4729b13f36d4713ce1162dc1d4425d44e1474078.patch
https://bugs.gentoo.org/829705

From: Ariadne Conill <ariadne@dereferenced.org>
Date: Mon, 26 Apr 2021 00:41:20 -0600
Subject: [PATCH] nvme-rpmb: Add limits.h due to missing PATH_MAX definition

Commit 21f40f38b introduced new use of PATH_MAX, but did not add the
limits.h header.  This resulted in nvme-cli failing to build on
ppc64le systems using the musl C library.

Signed-off-by: Ariadne Conill <ariadne@dereferenced.org>
[dwagner: removed nvme-topology.c bits]
Signed-off-by: Daniel Wagner <dwagner@suse.de>
--- a/nvme-rpmb.c
+++ b/nvme-rpmb.c
@@ -25,6 +25,7 @@
 #include <sys/socket.h>
 #include <linux/if_alg.h>
 #include <linux/socket.h>
+#include <limits.h>
 
 #include "nvme.h"
 #include "libnvme.h"
